staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -53,7 +54,8 @@ func NewAdmissionOptions() *AdmissionOptions {
 	options := &AdmissionOptions{
 		Plugins:                &admission.Plugins{},
 		PluginNames:            []string{},
-		RecommendedPluginOrder: []string{lifecycle.PluginName},
+		RecommendedPluginOrder: []string{lifecycle.PluginName, initialization.PluginName},
+		DefaultOffPlugins:      []string{initialization.PluginName},
```

Comment:
should we really switch this off?

Reply:
It's an alpha feature, so should be default off.

Reply:
don't we have feature gates for turning off alpha features by default?

Reply:
The story of initializers is a bit twisted. If the admission plugin is enabled, the feature gate will be enabled: https://github.com/kubernetes/kubernetes/pull/51436#issuecomment-325250675